cloudify_rest_client/plugins_update.py

```diff
@@ -0,0 +1,132 @@
+#########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+from cloudify_rest_client.responses import ListResponse
+
+
+class PluginsUpdate(dict):
+
+    def __init__(self, update):
+        self.update(update)
+
```

Comment:
force is missing

Reply:
Force is not saved under a plugin update, it's just to tell the manager to ignore other running updates.

Reply:
I think we need to because the use while looking at the history will want to know this

Reply:
I added it